src/app/key-concepts/natural-capital-and-climate-change/page.tsx

```diff
@@ -0,0 +1,197 @@
+                <div className="w-1.5 bg-sky" />
+                <div className="max-w-[510px] grow basis-0">
+                  Such actions are usually mutualistic –building natural capital increases carbon
+                  sequestration and vice-versa – but care is needed to guard against perverse
+                  outcomes where this may not occur.
+                </div>
+              </div>
+            </div>
+          </div>
+          <div className="w-full bg-sky bg-cover p-6 py-10 lg:bg-[url(/assets/climate-change-background-imperative.png)] lg:p-10">
+            <div className="flex flex-col gap-[25px]">
+              <div className="max-w-[790px] text-balance text-2xl lg:text-4xl">
+                It is imperative that businesses recognise and respond to this inter-dependence.
+              </div>
+              <div className="max-w-[790px]">
+                <p>
+                  To do so, they need to explicitly document their climate and nature-related
+                  dependencies and impacts, and understand the links between the two.
+                </p>
+                <p>
+                  Importantly, failure to consider and include natural capital in climate strategies
+                  will ultimately be ineffective in improving their climate position and expose the
+                  business to significant climate and nature-related risk.
+                </p>
+              </div>
+            </div>
+          </div>
+          <div className="flex w-full flex-wrap-reverse justify-between gap-10 lg:mb-20">
+            <div className="flex h-fit min-w-[394px] gap-6 text-grey-800">
+              <div className="w-1.5 min-w-1.5 bg-sky" />
+              <div className="max-w-[364px]">
+                Balancing economic growth with environmental sustainability and biodiversity
+                conservation is a pressing challenge that demands collaborative and proactive
+                solutions.
+              </div>
+            </div>
+            <div className="max-w-[540px] text-2xl lg:text-4xl">
+              Continuous research, policy development, and innovative business practices are crucial
+              to better incorporate natural capital into climate strategies.
+            </div>
+          </div>
+          <div className="relative mb-10 w-full overflow-hidden bg-black p-20 text-white lg:m-20">
+            <div className="relative flex flex-col items-center gap-[25px]">
+              <ImageCarousel
+                images={[
+                  {
+                    src: "/assets/product-story-1.png",
+                  },
+                  {
+                    src: "/assets/product-story-2.png",
+                  },
+                  {
+                    src: "/assets/product-story-3.png",
+                  },
+                  {
+                    src: "/assets/product-story-4.png",
+                  },
+                ]}
+                alt="Product Story"
```

Comment:
You don't have to add an alternative text if it's a decorative image. You can leave it empty: `""`.